pkg/webhook/validate_test.go

```diff
@@ -213,9 +213,8 @@ func TestValidatePolicyForNodeStateWithValidPolicy(t *testing.T) {
 		},
 	}
 	g := NewGomegaWithT(t)
-	ok, err := validatePolicyForNodeState(policy, state, NewNode())
```

Comment:
Test coverage is no longer complete because you have new error messages.

Reply:
Since we only adding errors for debugging/logging information, the test coverage that tests errors should be complete.

Reply:
We should test that the debugging/logging information is what we expect based on certain parameters.